spec/spec_helper.rb

```diff
@@ -11,35 +12,29 @@
 require 'napa'
 require 'napa/rspec_extensions/response_helpers'
 
+module NapaSpecClassHelpers
+  # Apparently Thor's last name is Odinson... who knew?!
+  # http://marvel.com/universe/Thor_(Thor_Odinson)
+  # http://www.infinitelooper.com/?v=4D7cPH7DHgA#/170;230
+  def tell_me_mr_odinson_what_good_is_a_phone_call_if_youre_unable_to_speak?
```

Comment:
Oh, apologies. I didn't mean to come off as rude. I just wasn't sure if you were being serious about the naming. Also, I thought it was poor form to push changes to other people's PRs. If that's you feel though, I suppose...